lib/utils.js

```diff
@@ -118,6 +118,21 @@ exports.filter = function(arr, fn){
 };
 
 /**
+ * Array#some
```

Comment:
Looks like we discussed this here: https://github.com/mochajs/mocha/issues/1330

Want to revisit and get a small lodash build going?